### PR TITLE
Add banners about a free account

### PR DESCRIPTION
**Pull request by gorzelinski**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Add banners about a free account and remove properties from guides` snippets.

---

### Additional information

This PR also contains other small fixes.

### CODE REVIEW

File: docs/getting-started/integrations-cdn/angular.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ All available Angular versions are [listed on npm](https://www.npmjs.com/package
 
 ## Quick start
 
+<info-box>
```
Comment (gorzelinski) [inline docs/getting-started/integrations-cdn/angular.md:100]: I experimented with this callout a bit. I would probably leave the placement as it is. I tried different text versions, and "You need an account" sounds slightly less harsh than "You have to do something." The yellow color also seemed too harsh.

File: docs/getting-started/integrations-cdn/laravel.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ This guide assume you have a Laravel project. You can create a basic Laravel pro
 
 ## Using from CDN
 
+<info-box>
+	To use our CDN services, you need [a free account](https://portal.ckeditor.com/checkout?plan=free).
```
Comment (gorzelinski) [inline docs/getting-started/integrations-cdn/laravel.md:22]: I'm assuming the link will work after releasing the portal?

File: docs/getting-started/integrations-cdn/angular.md (hunk 1 above)
Comment (filipsobol) [reply docs/getting-started/integrations-cdn/angular.md:100]: WDYT about this?

```markdown
To use our CDN services, you need to [create a free account](https://portal.ckeditor.com/checkout?plan=free).
```

File: docs/getting-started/integrations-cdn/laravel.md (hunk 2 above)
Comment (Witoso) [reply docs/getting-started/integrations-cdn/laravel.md:22]: yes, you can check the flow with ckeditor-dev.com

File: docs/getting-started/integrations-cdn/angular.md (hunk 1 above)
Comment (gorzelinski) [reply docs/getting-started/integrations-cdn/angular.md:100]: Maybe the key is the simplicity. How about now?

**Pull request merged**